packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -87,7 +93,8 @@ export class VaRadio {
   @Listen('keydown')
   handleKeyDown(event: KeyboardEvent) {
     const currentNode = event.target as HTMLVaRadioOptionElement;
-    const radioOptionNodes = getSlottedNodes(this.el, 'va-radio-option');
+    const radioOptionNodes = (getSlottedNodes(this.el, 'va-radio-option') as HTMLVaRadioOptionElement[])
+      .filter(node => !node.disabled);
```

Comment:
USWDS v3 provides a "disabled" variant for options and this modification is to handle navigating through the options so that it skips over those that are identified as disabled.